zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/migration/MigrateMessageSubscriptionTest.java

```diff
@@ -204,7 +205,199 @@ public void shouldCorrelateMessageSubscriptionAfterMigration() {
         .hasCorrelationKey("key1");
   }
 
-  // TODO - Test that interrupting status change is not allowed
+  @Test
+  public void shouldMigrateToInterruptingStatus() {
```

Comment:
🔧 Let's show that the functionality works as expected after migrating:
- publish a message
- see that the task is terminated (interrupting), or unaffected (non-interrupting)